SU-SWS/react_paragraphs
# D8CORE-3263 Improve the media library widget with cardinality above 1

# READY FOR REVIEW

# Summary
- fix removing selected media items when there are more than 1
- wrap media items when there are more than 4 or so.
- A later ticket was added to add sorting ability https://stanfordits.atlassian.net/browse/D8CORE-3380

# Need Review By (Date)
- 1/15

# Urgency
- medium

# Steps to Test
1. use the steps in https://github.com/SU-SWS/stanford_profile/pull/347
2. add a few images to the widget
3. add the same images again so that you have duplicates in the field (this is allowed in core as well)
4. remove the first duplicate
5. verify it removes the correct image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
